avocado/utils/software_manager/backends/apt.py

```diff
@@ -19,20 +19,28 @@ class AptBackend(DpkgBackend):
     Debian based distributions, such as Ubuntu Linux.
     """
 
-    def __init__(self):
+    def __init__(self, session=None):
```

Comment:
I was not able to manually test the apt backend, so please review this carefully.